dist-git/dist-git.spec

```diff
@@ -92,13 +94,14 @@ cd -
 
 
 %pre
+%if 0%{?fedora} && 0%{?fedora} <= 41
```

Comment:
This is weird, I thought we got the code correctly for Mock:
https://github.com/rpm-software-management/mock/blob/main/mock/mock.spec#L10

We should special-case EPEL9, but are you sure about Fedora 40?

Reply:
> Do we care about suse and mageia? I am not aware of any usage.

Me neither..

>  I can add the rhel part.

It would be nice to keep RHEL dist-git (server package) variant working for a while (if it costs almost no efforts); at least as an alternative to the Konflux lookaside (or dist-git?) implementation (I have no link/reference yet, but @mikem23 will discuss this with Fedora community in the following days)